src/core/vectortile/qgsvectortiledataprovider.h

```diff
@@ -114,6 +120,17 @@ class CORE_EXPORT QgsVectorTileDataProvider : public QgsDataProvider
      */
     virtual QString sourcePath() const = 0;
 
+
+    /**
+     * Returns the list of source paths for the data.
+     * \since QGIS 3.40
+     */
+    virtual QgsStringMap sourcePaths() const
+    {
+      return {{QString(), sourcePath()}};
```

Comment:
Can we add some spacing for these map declaration? 
```suggestion
      return { { QString(), sourcePath() }  };
```